cmd/describe/ingress/cmd.go

```diff
@@ -0,0 +1,87 @@
+package ingress
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/ingress"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+const (
+	use     = "ingress"
+	short   = "Show details of the specified ingress within cluster"
+	example = `rosa describe ingress <ingress_id> -c mycluster`
+)
+
+// Regular expression to used to make sure that the identifier given by the
+// user is safe and that it there is no risk of SQL injection:
+var ingressKeyRE = regexp.MustCompile(`^[a-z0-9]{3,5}$`)
+
+func NewDescribeIngressCommand() *cobra.Command {
+	options := NewDescribeIngressUserOptions()
+	cmd := &cobra.Command{
+		Use:     use,
+		Short:   short,
+		Example: example,
+		Run:     rosa.DefaultRunner(rosa.RuntimeWithOCM(), DescribeIngressRunner(options)),
+		Args: func(_ *cobra.Command, argv []string) error {
+			if len(argv) != 1 {
+				return fmt.Errorf(
```

Comment:
do we need this check here? Later in the runner we check for length of args, might could throw that in there along with that logic to keep similar logic together

Reply:
We also check in `options.Bind` for an ingress ID, so that code path will probably never be reached